packages/frontend/src/components/Form/Input.tsx

```diff
@@ -22,13 +22,21 @@ export const Input = ({ bid, setBid, isBadAmount }: InputProps) => {
   const initialInputValue = bid.isZero() ? '' : formatEther(bid)
   const [inputValue, setInputValue] = useState(initialInputValue)
 
-  const setValue = (value: string) => {
+  const onChange = (value: string) => {
     if (!numberInputRegex.test(value)) {
       return
     }
-    setInputValue(value)
     if (value !== '') {
-      setBid(parseEther(value))
+      const formattedValue = value.replace(',', '.')
+      setInputValue(formattedValue)
+      setBid(parseEther(formattedValue))
+    }
```

Comment:
```suggestion
    }
    setInputValue(value)
    setBid(BigNumber.from(0))
```
We need to handle `value === ''`, otherwise it won't let the user delete the last remaining digit. We should do something to strip leading zeroes too, probably.